demo/demo_frame_graphs.hpp

```diff
@@ -23,17 +24,25 @@ namespace fg_manager
 
 	inline void Setup(wr::RenderSystem& rs, util::Delegate<void()> imgui_func)
 	{
-		// Raytracing
+		// Ray tracing
 		{
 			auto& fg = frame_graphs[(int)PrebuildFrameGraph::RAYTRACING];
 			fg = new wr::FrameGraph(3);
 
 			wr::AddBuildAccelerationStructuresTask(*fg);
 			wr::AddRaytracingTask(*fg);
 			wr::AddAccumulationTask(*fg);
+			
+			// Copy the scene render pixel data to a buffer that is CPU-accessible
+			wr::AddRenderTargetReadBackTask<wr::RaytracingData>(*fg, std::nullopt, std::nullopt);
```

Comment:
We probably want to put this into a different frame graph. We don't want to read-back when running as standalone.